### PR TITLE
DEV2-3412: Remove chat control buttons

### PR DESCRIPTION
**Pull request by ofekby**

### CODE REVIEW

File: Common/src/main/java/com/tabnineCommon/chat/actions/TabnineActionsGroup.kt [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
     companion object {
-        fun create(browser: ChatBrowser, isEnterprise: Boolean): TabnineActionsGroup {
+        fun create(browser: ChatBrowser): TabnineActionsGroup {
             val group = TabnineActionsGroup()
-            group.add(ClearConversationAction(browser))
-            group.add(NewConversationAction(browser))
-            group.add(ChatHistoryAction(browser))
-            group.add(SubmitFeedbackAction(browser))
-            if (!isEnterprise) {
-                group.add(ChatSettingsAction(browser))
```
Comment (yanir-codota) [inline Common/src/main/java/com/tabnineCommon/chat/actions/TabnineActionsGroup.kt:21]: what is this action?

File: Common/src/main/java/com/tabnineCommon/chat/actions/TabnineChatToolWindowAction.kt [hunk 2]
```diff
@@ -14,11 +12,4 @@ abstract class TabnineChatAction(
     description: String? = text,
     icon: Icon? = null
 ) :
-    AnAction(text, description, icon) {
-    fun sendMessage(message: TabnineActionRequest) {
-        val messageJson = DependencyContainer.instanceOfGson().toJson(message)
-
-        Logger.getInstance(javaClass).info("Sending message: $messageJson")
-        browser.jbCefBrowser.cefBrowser.executeJavaScript("window.postMessage($messageJson, '*')", "", 0)
-    }
```
Comment (yanir-codota) [inline Common/src/main/java/com/tabnineCommon/chat/actions/TabnineChatToolWindowAction.kt:23]: why was this removed?
is there no event for debug tool opening?

File: Common/src/main/java/com/tabnineCommon/chat/ChatFrame.kt [hunk 3]
```diff
@@ -105,11 +102,7 @@ class ChatFrame(private val project: Project, private val messagesRouter: ChatMe
             return
         }
 
-        val stateResponse =
-            DependencyContainer.instanceOfBinaryRequestFacade().executeRequest(StateRequest())
```
Comment (yanir-codota) [inline Common/src/main/java/com/tabnineCommon/chat/ChatFrame.kt:109]: why is this removed?

**Review by yanir-codota — COMMENTED** [review]
All in all looks good, Had few questions.
Also, please add a screenshot of how the result will look like.

**Review by amirbilu — APPROVED** [review]
